pkg/autodiscover/autodiscover.go

```diff
@@ -190,22 +195,23 @@ func namespacesListToStringList(namespaceList []configuration.Namespace) (string
 func getOpenshiftVersion(oClient clientconfigv1.ConfigV1Interface) (ver string, err error) {
 	var clusterOperator *configv1.ClusterOperator
 	clusterOperator, err = oClient.ClusterOperators().Get(context.TODO(), "openshift-apiserver", metav1.GetOptions{})
-	// error here indicates logged in as non-admin, log and move on
 	if err != nil {
 		switch {
-		case kerrors.IsForbidden(err), kerrors.IsNotFound(err):
-			logrus.Infof("OpenShift Version not found (must be logged in to cluster as admin): %v", err)
-			err = nil
+		case kerrors.IsNotFound(err):
+			logrus.Warnf("Unable to get ClusterOperator CR from openshift-apiserver. Running in a non-OCP cluster.")
+			return "", nil
```

Comment:
Add const `none` or `0.0.0+nil` in case of not found.

Reply:
Done.